src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
```

Comment:
your whole validation config does not work now, 1 thing is missing, adding annotations in dtos is not enough